rasa/core/actions/action.py

```diff
@@ -584,19 +584,122 @@ def action_response_format_spec() -> Dict[Text, Any]:
 
         Used for validation of the response returned from the
         Action endpoint."""
-        return {
+        entities = {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "start": {"type": "integer"},
+                    "end": {"type": "integer"},
+                    "name": {"type": "string"},
+                    "confidence": {"type": "number"},
+                    "extractor": {},
+                    "value": {"type": "object"},
+                },
+            },
+        }
+        schema = {
             "type": "object",
             "properties": {
                 "events": {
                     "type": "array",
                     "items": {
                         "type": "object",
-                        "properties": {"event": {"type": "string"}},
+                        "properties": {
+                            "event": {"type": "string"},
+                            "timestamp": {},
```

Comment:
don't we know the timestamp is `Optional[float]`?

Reply:
yup, amended now.